k2/addons/k2-power/powerswitch/__init__.py

```diff
@@ -48,7 +48,7 @@
     """)
 
 POWER_SWITCH_POWER_STATE = MessageId(
-    'POWER_SWITCH_POWER_STATUS', """\
+    'POWER_SWITCH_POWER_STATE', """\
```

Comment:
Feel free to merge this specific change without further review. Since it is independent of the sispm implementation it can be handled within its own commit. See also my general comment about making sispm its own addon. In that context it makes more sense to handle this specific change as a separate thing.